NEWS

```diff
@@ -1,3 +1,7 @@
+* Add access to the new `direction` argument of `scales::brewer_pal` and
```

Comment:
Sorry I think I was unclear bfore, this should be the first bullet under the version heading